src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc

```diff
@@ -125,7 +118,7 @@ std::shared_ptr<FiniteDiffGroundTempsModel> FiniteDiffGroundTempsModel::FiniteDi
         }
     }
 
-    if (found && !ErrorsFound) {
+    if (found) {
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc:128]:(style),[knownConditionTrueFalse],Condition '!ErrorsFound' is always true